packages/terra-menu/src/MenuDivider.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Is this component ever expected to be used outside of Menu? I am curious if it should be an internal component.